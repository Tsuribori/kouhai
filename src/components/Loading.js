import React from 'react';
import PropTypes from 'prop-types';
import { withStyles } from '@material-ui/core/styles';
import { Grid, CircularProgress } from '@material-ui/core';

const styles = theme => ({
  padded: {
    paddingTop: theme.spacing.unit * 5,
  }
});

function Loading(props) {
  return (
    <Grid container
      justify="center"
      className={props.classes.padded}
    >
      <Grid item>
        <CircularProgress/>
      </Grid>
    </Grid>
  );
}

Loading.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(styles)(Loading);
