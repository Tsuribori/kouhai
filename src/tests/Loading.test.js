import React from 'react';
import './TestSetUp.js';
import Loading from '../components/Loading.js';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('Test Loading snapshot', () => {
  const tree = shallow(<Loading />).dive();
  expect(toJson(tree)).toMatchSnapshot();
});
