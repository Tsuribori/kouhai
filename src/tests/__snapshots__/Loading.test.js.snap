// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Loading snapshot 1`] = `
<WithStyles(Grid)
  className="Loading-padded-1"
  container={true}
  justify="center"
>
  <WithStyles(Grid)
    item={true}
  >
    <WithStyles(CircularProgress) />
  </WithStyles(Grid)>
</WithStyles(Grid)>
`;
